manifests/charts/base/templates/validatingwebhookconfiguration.yaml

```diff
@@ -9,10 +9,14 @@ metadata:
 webhooks:
   - name: validation.istio.io
     clientConfig:
+      {{- if and (.Values.global.remotePilotAddress) (.Values.global.centralIstiod) }}
+      url: https://{{ .Values.global.remotePilotAddress }}:443/validate
+      {{- else }}
```

Comment:
I think this is too narrow - how about: add a new setting in base/values.yaml ( since this is clearly not a global ), with the key "validationURL". 

If set - have validation use it, otherwise - old behavior. 

Setting validation to a custom URL is not tied in any way to 'centralIstio' - we can very well use it in any istio configuration. And it should allow the entire URL to be customized - no need to hardcode /validate.

Reply:
I was trying to avoid adding new configs. If you think there is value to use this beyond central Istiod scenario, I'm happy to add validationURL and mutatingURL for you.

Reply:
updated